roles/dnsmasq/tasks/manage_host.yml

```diff
@@ -14,72 +14,100 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Ensure we have the right data and type
```

Comment:
do we also need to check if they are using the 'old' vars and let them know? or this one will catch it?

Reply:
if they use the old one, they won't provide the new one - so crash